modules/issue_tracker/php/my_issue_tracker.class.inc

```diff
@@ -147,19 +148,26 @@ class NDB_Menu_Filter_Form_My_Issue_Tracker extends NDB_Menu_Filter_Form
     }
 
     /**
-     * Setup $this->tpl_data for use by Smarty
+     * Sets up the smarty menu filter items for the conflict resolver
```

Comment:
I think this comment is wrong.. this is the issue tracker, not the conflict resolver